doc/build-recorder-schema.ttl

```diff
@@ -0,0 +1,65 @@
+# Copyright (C) 2023 Fotios Valasiadis
+# Copyright (C) 2023 Alexios Zavras
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+@prefix b: <http://example.org/build-recorder#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Types
+b:process a rdfs:Class .
+b:file a rdfs:Class .
+
+# Process Properties
+b:pid a rdf:Property ;
```

Comment:
`rdf` namespace?!? different one?

Reply:
@zvr Well according to the results of my research while searching for how to solve this issue, I came upon this:

> The correct namespaces to include in the RDF/Turtle file are rdf and rdfs. The rdf namespace is used for the core RDF vocabulary, while the rdfs namespace is used for the RDF Schema vocabulary.

Is said information invalid? Do I just use rdfs everywhere?